src/Renderer/LayeredMaterial.js

```diff
@@ -253,6 +253,12 @@ class LayeredMaterial extends THREE.RawShaderMaterial {
     getElevationLayer() {
         return this.layers.find(l => l.id === this.elevationLayerIds[0]);
     }
+
+    setElevationScale(scale) {
+        if (this.elevationLayerIds.length > 0) {
```

Comment:
could you replace by  `if (this.elevationLayerIds.length)`

Reply:
done